metrics/cli/docs/features/diagrams/yaml_config_parser_class_diagram.puml

```diff
@@ -0,0 +1,52 @@
+@startuml yaml_config_parser_class_diagram
+
+package cli {
+    package updater {
+        package config {
+            package factory {
+                class UpdateConfigFactory {
+                    - _fileHelper : FileHelper
+                    - _configParser : UpdateConfigParser
+                    + create(String configPath) : UpdateConfig
+                }
+            }
+
+            package model {
+                class UpdateConfig {
+                    + firebaseConfig : FirebaseConfig
+                    + sentryConfig : SentryConfig
+                }
+
+                class FirebaseConfig {
+                    + authToken : String
+                    + projectId : String
+                    + googleSignInClientId : String
+                }
+
+                class SentryConfig {
+                    + authToken : String
+                    + organizationSlug : String
+                    + projectSlug : String
+                    + projectDsn : String
+                    + releaseName: String
+                }
+            }
+
+            package parser {
+                class UpdateConfigParser {
+                    - {static} _parser : YamlMapParser
+                    + parse(String configYaml) : UpdateConfig
+                }
+            }
+        }
+    }
+}
+
+UpdateConfigParser --> UpdateConfig : uses
+UpdateConfigFactory --> UpdateConfigParser : uses
+UpdateConfigFactory --> UpdateConfig : creates
+
+FirebaseConfig -up-o UpdateConfig
+SentryConfig -up-o UpdateConfig
```

Comment:
🏗️ 
Should it be a composition instead of aggregation? Also, how about adding some numbers to indicate instance relation (one-to-one, one-to-many)?